README.md

```diff
@@ -395,7 +395,8 @@ supported by this (and past) versions of this repo (go-github).
 Versions prior to 48.2.0 are not listed.
 
 | go-github Version | GitHub v3 API Version |
-| ----------------- | --------------------- |
+|-------------------| --------------------- |
```

Comment:
Please do not modify version information in PRs... you can revert the changes to this file.
Versions are maintained and controlled by the repo owners.  Thanks.